media-libs/liblqr/liblqr-0.4.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Easy to use C/C++ seam carving library"
+HOMEPAGE="
+	https://liblqr.wikidot.com/
+	https://github.com/carlobaldassi/liblqr
+"
+# NOTE: github also offers tarballs but wikidot uses bz2
+SRC_URI="https://liblqr.wikidot.com/local--files/en:download-page/${PN}-1-${PV}.tar.bz2 -> ${P}.tar.bz2"
+S="${WORKDIR}/${PN}-1-${PV}"
+
+LICENSE="|| ( GPL-3 LGPL-3 )"
+SLOT="3.2.3"
```

Comment:
Why are you adding a slot? That's pretty unusual?

Reply:
Like, we can't just slot something for the hell of it -- it has to be parallel installable with the previous version. We also need to check consumers are ready for it.